amazon-location-route-planner/README.md

```diff
@@ -0,0 +1,51 @@
+# Amazon Location Route Planner
+
+This sample app demonstrates Amazon Location Service's routing functionality in a web application. It finds the fastest route using a number of parameters such as different modes of transportation (car, truck, and walking), departure time, avoidances (ferries and tolls), and weight and size limitations for trucks. This sample app is built with Vue.js, MapLibre GL JS, and Tailwind CSS.
+
+## Requirements
+
+- Create an [AWS account](https://portal.aws.amazon.com/billing/signup).
+- Install [Node.js](https://nodejs.org/en/download/) and [npm](https://docs.npmjs.com/getting-started).
+
+## Create resources
+
+Click the button below to create the necessary AWS resources for this sample app to run. It will open the AWS Management Console and initiate the CloudFormation template deployment process.
+
+[![Launch Stack](https://amazon-location-blog-assets.s3.us-west-2.amazonaws.com/cfn-launch-stack-button.svg)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/quickcreate?stackName=amazon-location-route-planner&templateURL=https://amazon-location-blog-assets.s3.us-west-2.amazonaws.com/amazon-location-route-planner/template.yml)
+
+Once the deployment process is complete, go to the `Outputs` section to get the Cognito Identity Pool ID.
+
+## Configure
+
+Open `src/config.js` and use the CloudFormation stack outputs to enter your Cognito Identity Pool ID.
+
+## Run
+
+Install dependencies and run the application.
+
+```
+npm install
+npm run dev
```

Comment:
This opens on port 5173 which is not compatible with AWS Cloud9. All our other samples specify port 8080 in the vite config

Reply:
fixed with [a7325b3](https://github.com/aws-samples/amazon-location-samples/pull/247/commits/a7325b34f122398ecc77f9681b70443cfe730e2f)